### ISSUE TITLE
How to add a relationship between form model?

### ISSUE THREAD

**Issue by b-jam** [samluescher/django-form-designer#89]
How can I render a form inside my own template. There are no templatetags I can see.

in my models.py:

from form_designer.contrib.cms_plugins.form_designer_form.models import FormDefinition as Form
class MyObject:
    form = models.ForeignKey(Form, null=True, blank=True)

in my templates/myobject.htm;
# what tag should I {% load %}?

{% if myobject.form %}
# how to render form here???

{% endif %}

**Comment by b-jam** [comment]
I tried 
                {% for item in data %}
                {% if item.label %}{{ item.label }}{% else %}{{ item.name }}{% endif %}: {{ item.value|friendly|safe }}
                {% endfor %}

from the templates but doesn't work.

**Comment by samluescher** [comment]
First you need to instantiate a form instance based on your FormDefinition instance.

```
from form_designer.forms import DesignedForm
form = DesignedForm(form_definition)
```

The DesignedForm instance is rendered like a regular Django form. You can render it manually, or use something like form.as_table. See https://docs.djangoproject.com/en/dev/topics/forms/#working-with-form-templates.

Check out views.py::process_form() to see how it handles form submissions. It's pretty standard Django stuff, with the exception of how the form is defined.

**Comment by b-jam** [comment]
Thanks, I'm new to django but your reply gave me everything I needed to
know :D

On 5 November 2014 10:15, Samuel Luescher notifications@github.com wrote:

> Closed #89 https://github.com/samluescher/django-form-designer/issues/89
> .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/samluescher/django-form-designer/issues/89#event-188306073
> .

## 

Cheers,
Ben
